content/docs/command-reference/config.md

```diff
@@ -19,32 +19,39 @@ You can query/set/replace/unset DVC configuration options with this command. It
 takes a config option `name` (a config section and a key, separated by a dot)
 and its `value` (any valid alpha-numeric string generally).
 
-This command reads and updates the DVC configuration files. By default (if none
-of `--local`, `--global`, or `--system` is provided) a project's config
-(`.dvc/config`) file is read or modified. This file is by default meant to be
-tracked by Git and should not contain sensitive and/or user-specific information
-(passwords, SSH keys, etc). Use `--local`, `--global`, or `--system` command
-options (flags) instead to override project's settings, for sensitive, or
-user-specific settings.
-
-If the config option `value` is not provided and the `--unset` command option is
-not used, this command returns the current value of the config option, if found
-in the corresponding config file.
+If the config option `value` is not provided (and without `--unset`), this
+command returns the current value of the config option, if found in the
+corresponding config file.
+
+This command reads and updates the DVC configuration files. By default, the
+regular project's config file in `.dvc/config` is read or modified. This file is
+meant to be tracked by Git and should not contain sensitive and/or user-specific
+information (passwords, SSH keys, etc). Use the `--local` command option (flag)
+instead, to set (or override) secrets. `--global` and `--system` are also
+available to set config options for multiple projects and users, respectively.
+
+| Option     | Priority | Linux/Mac location     | Windows location                                          |
+| ---------- | -------- | ---------------------- | --------------------------------------------------------- |
+| `--local`  | 1        | `.dvc/config.local`    | `.dvc/config.local`                                       |
+| None       | 2        | `.dvc/config`          | `.dvc/config`                                             |
+| `--global` | 3        | `~/.config/dvc/config` | `%LocalAppData%/iterative/dvc/config`                     |
```

Comment:
This is not correct for Mac (for global/system at least)

Reply:
Oh can you help me confirm what those are? I don't have a Mac with me rn, sry

Reply:
Removed "Mac" for now, merging this, but will update ASAP as I find out the right locations.